test/receiver_test.exs

```diff
@@ -12,10 +12,37 @@ defmodule ReceiverTest do
 
   doctest Receiver
 
+  describe "supervsion" do
+    test "successfully starts up under a supervisor using a child spec" do
```

Comment:
We are not asserting on anything in this test. I think it may be more better to do in integration test, where we can start sender and validate against it. Or we simply can assert on process state. What do you think?